starkinfra/creditnotepreview/__creditnotepreview.py

```diff
@@ -0,0 +1,78 @@
+from starkcore.utils.subresource import SubResource
+from starkinfra.utils import rest
+from ..creditnote.invoice.__invoice import Invoice, _resource as _invoice_resource
+from starkcore.utils.api import from_api_json
+
+
+class CreditNotePreview(SubResource):
+    """# CreditNotePreview object
+    CreditNotePreview are used to preview CCB contracts between you and your customers with
+    selected table type.
```

Comment:
A CreditNotePreview is used to preview a CCB contract between the borrower and lender with a specific table type.